en/docs/edit-the-code/code-actions.md

```diff
@@ -1 +1,70 @@
-# Code actions
\ No newline at end of file
+# Code actions
+
+## Types of Code actions
+
+### QuickFixes
+
+QuickFixes are suggested based on the diagnostic displayed at a given cursor position.
+
+#### Code Actions for Create Variables
+
+The code action types below are available for creating a variable.
+- `Create variable`: Create a variable for an expression where the `Variable Assignment Required` diagnostic is present.
+- `Create variable and type guard`: Create a type guard to handle the error gracefully when the `Variable assignment Required` diagnostic is present.
+- `Create variable and check error`: Add a check expression when the `Variable assignment Required` diagnostic is present.
+- `Create variable with type`: Create a variable for a remote method expression where the `Cannot infer the 'typedesc' argument for parameter` diagnostic is present.
+- `Ignore return value`: Ignore the return value with the `_` where the `Variable Assignment Required` diagnostic is present.
+
+#### Code Actions for Union Types
+
+The code actions below are available for union-type variables.
+- `Add check error`: Add a check statement when there is an error union.
+
+#### Code Actions for Imports
```

Comment:
Let's name this section `Quick fixes for imports`. Same for the following topics under quick fixes section. Let's start the topic as `Quick fixes for xxx`